internal/rpmmd/repository.go

```diff
@@ -392,10 +392,9 @@ func NewRPMMD(cacheDir, dnfJsonPath string) RPMMD {
 	}
 }
 
-func (repo RepoConfig) toDNFRepoConfig(rpmmd *rpmmdImpl, i int, arch, releasever string) (dnfRepoConfig, error) {
-	id := strconv.Itoa(i)
+func (repo RepoConfig) toDNFRepoConfig(rpmmd *rpmmdImpl, arch, releasever string) (dnfRepoConfig, error) {
 	dnfRepo := dnfRepoConfig{
-		ID:             id,
+		ID:             repo.Name,
```

Comment:
Similarly to above, are we guaranteed that the Name will be unique? (and an appropriate identifier)